api/src/main/java/org/pmiops/workbench/interceptors/InterceptorUtils.java

```diff
@@ -46,7 +46,7 @@ public static Method getControllerMethod(HandlerMethod handlerMethod) {
    *
    * <p>For cloud task endpoint has cloudTask tag generated by swagger.
    */
-  public static boolean isCloudTaskRequest(ApiOperation apiOp) {
```

Comment:
This is also related to the annotation change:
https://support.intershop.com/kb/index.php/Display/2914L4